python/paddle_fl/mpc/data_utils/op_extra_desc.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module add extra input/output variable for some ops, including
+relu, relu_grad, pool2d, pool2d_grad.
+"""
+import abc
+import six
+
+ops_to_add_extra = ['relu', 'relu_grad', 'pool2d', 'pool2d_grad']
+
+def add_extra_desc(op, block):
+    if op.type not in ops_to_add_extra:
```

Comment:
The if...else... clause has redundant checks, i.e., even if op is relu, the other checks will be run, which is expected not.

use dict to store the op_desc instance, and get rid of the if...else... clauses.
e.g.,  ops_to_add_extra = {‘relu’:ReluExtraDesc(), 'relu_grad':ReluGradExtraDesc, ...}

then get the instance directly by the key:
op_desc_inst = ops_to_add_extra.get(op.type)
if op_desc_inst:
    op_desc_inst.add_extra_desc(op, block)